Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup \
 
 # Install prerequisites
 RUN set -x && \
-  apt-get update && apt-get -y upgrade && \
```

Comment:
From https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/: "You should avoid RUN apt-get upgrade or dist-upgrade"